corehq/motech/repeaters/tests/test_models.py

```diff
@@ -690,14 +698,16 @@ def new_record(self, next_check=before_now, state=State.Pending, domain="test"):
             state=state,
         )
 
-    def make_records(self, n):
+    def make_records(self, n, state=State.Pending):
         now = timezone.now() - timedelta(seconds=10)
+        is_pending = state in [State.Pending, State.Fail]
```

Comment:
See response to your other comment.

Reply:
I think I can resolve the ambiguity with a comment on the `State` enum. It's not perfect, but it would at least be something to point to when there are questions. I'll do that in a separate PR.